src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/ResourceGroup.kt

```diff
@@ -0,0 +1,7 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
```

Comment:
Use vals, not vars, I think.

Reply:
Good point. I was at one point under the false assumption that data classes had to contain vars, not sure where I got that idea.